website/docs/reference/resource-configs/fabric-configs.md

```diff
@@ -3,103 +3,910 @@ title: "Microsoft Fabric DWH configurations"
+
+        # only if `data_type` is 'int64'
+        "range": {
+            "start": <integer>,
+            "end": <integer>,
+            "interval": <integer>,
+        }
+    },
+
+    # auto-refresh options
+    [enable_refresh](#auto-refresh)= true | false,
+    [refresh_interval_minutes](#auto-refresh)=<float>,
+    [max_staleness](#auto-refresh)="<interval>",
+
+    # additional options
+    [description](/reference/resource-properties/description)="<description>",
+    [labels](#specifying-labels)={
+        "<label-name>": "<label-value>",
+    },
+    [hours_to_expiration](#acontrolling-table-expiration)=<integer>,
+    [kms_key_name](##using-kms-encryption)="<path_to_key>",
+) }}
 ```
 
 </File>
 
-## Snapshots
+</TabItem>
+
+</Tabs>
+
+Many of these parameters correspond to their table counterparts and have been linked above.
+The set of parameters unique to materialized views covers [auto-refresh functionality](#auto-refresh).
+
+Learn more about these parameters in BigQuery's docs:
+- [CREATE MATERIALIZED VIEW statement](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#create_materialized_view_statement)
+- [materialized_view_option_list](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#materialized_view_option_list)
+
+### Auto-refresh
 
-Columns in source tables can not have any constraints.
-If, for example, any column has a `NOT NULL` constraint, an error will be thrown.
+| Parameter                    | Type         | Required | Default | Change Monitoring Support |
+|------------------------------|--------------|----------|---------|---------------------------|
+| `enable_refresh`             | `<boolean>`  | no       | `true`  | alter                     |
+| `refresh_interval_minutes`   | `<float>`    | no       | `30`    | alter                     |
+| `max_staleness` (in Preview) | `<interval>` | no       | `none`  | alter                     |
 
-## Indexes
+BigQuery supports [automatic refresh](https://cloud.google.com/bigquery/docs/materialized-views-manage#automatic_refresh) configuration for materialized views.
+By default, a materialized view will automatically refresh within 5 minutes of changes in the base table, but not more frequently than once every 30 minutes.
+BigQuery only officially supports the configuration of the frequency (the "once every 30 minutes" frequency);
+however, there is a feature in preview that allows for the configuration of the staleness (the "5 minutes" refresh).
+dbt will monitor these parameters for changes and apply them using an `ALTER` statement.
 
-Indexes are not supported by Microsoft Fabric Synapse Data Warehouse. Any Indexes provided as a configuration is ignored by the adapter.
+Learn more about these parameters in BigQuery's docs:
+- [materialized_view_option_list](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#materialized_view_option_list)
```

Comment:
this shouldn't be flagged as a typo. 

weird, the parameter names in the table above aren't getting flagged. oh i see the difference! the square brackets